features/web-animations.yml

```
@@ -1,6 +1,70 @@
 name: Web animations
-description: The web animations API allows you to animate and synchronize the animations of DOM elements.
+description: The web animations JavaScript API allows you to animate and synchronize the animations of DOM elements.
```

```suggestion
description: The web animations API programmatically animates elements over time, and allows you to synchronize the animations of multiple elements.
```

We don't really prefix API with JavaScript in other features, even when they are, indeed, JS-based. I know that you probably felt it was important to add it here, to make it clear that this was separate from CSS and SVG animations. What about using the word "programmatically", to differentiate from "declaratively"?
I'm not super sold on it. Let me know your thoughts.

I also slightly rephrased because the existing sentence felt a bit hard to parse to me (I was probably the one who wrote it originally 😄).

And I, again, added the time concept here. I remember, years ago, when the WAAPI was supposed to grow the ability to support other types of timelines, such as scroll position (I still see some verbiage about this in the spec). I guess this never happened, and we now have scroll-driven animations purely in CSS defined in a separate spec, and not doable with WAAPI. So it's probably a good thing to say that these are time-based only.

---

Looking at this again, "Web Animations API" is a proper noun, and probably should be capitalized. I do think that JavaScript is an important word to include here, as that is both how this is used, and the key differentiator with other animation techniques. Those do mention their language, and we shouldn't assume JavaScript as the default language. 

How does this look? I agree the time factor is important to include.

```suggestion
description: The Web Animations API programmatically animates elements over time, and allows you to synchronize the animations of multiple elements in JavaScript.
```